application-task-ui/src/main/resources/TaskManager/AdministrationAddProject.xml

```diff
@@ -58,9 +58,9 @@
       &lt;/span&gt;
     &lt;/form&gt;
   {{/html}}
-  #set($columns = ['doc.name', 'doc.date', 'doc.author', '_actions'])
+  #set($columns = ['project', 'doc.date', 'doc.author', '_actions'])
```

Comment:
Using the `project` is also good. I was thinking more of `doc.title` while also setting the title higher up in the code. `$document.setTitle("$project")`

Reply:
This way, visiting the page will not display `Project_`. Also, the changes should be done to all the other files as well: `AdministrationAddStatus`, `AdministrationAddSeverity`.

Reply:
Just to clarify, you propose to replace mine change `project` with `doc.title` on each livetable for Project, Status and Priority? Then on other two livetable replace `doc.name` with `doc.title`?

Reply:
If I set `#set($discard = $document.setTitle("$severity"))` with further replacement in livetable `doc.name` into `doc.title`, the info box appears

"(<sup>1</sup>) Some pages have a computed title. Filtering and sorting by title will not work as expected for these pages."

which just informative because in livetable `'filterable' : false` but it will still present no matter what. 

![image](https://github.com/xwikisas/application-task/assets/56109799/20550f5e-f49b-4b50-bffe-d365c53460e4)

That's why if I change `doc.title` on `$severity` it displays nice without info message and <sup>1</sup> in title

![firefox_Fj8TVe0r6n](https://github.com/xwikisas/application-task/assets/56109799/5ba026e4-3293-44f0-b30b-3a2790a13ff9)